src/components/VirtualizedCombobox.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState, useCallback } from 'react';
+import {
+  Command,
+  CommandEmpty,
+  CommandGroup,
+  CommandInput,
+  CommandItem,
+  CommandList,
+} from './ui/command';
+import { Popover, PopoverContent, PopoverTrigger } from './ui/popover';
+import { Button } from './ui/button';
+import { useVirtualizer } from '@tanstack/react-virtual';
+import { Check as CheckIcon, ChevronsUpDown } from 'lucide-react';
+import { cn } from '@/lib/utils';
+import { SymbolList } from '@/stores/fetchSymbolList';
+import { ScrollArea } from './ui/scroll-area';
+
+type VirtualizedComboboxProps = {
+  options: SymbolList[];
+  placeholder: string;
+  selectedOption: string;
+  onSelect: (symbol: string) => void;
+  height?: string;
+};
+
+export const VirtualizedCombobox: React.FC<VirtualizedComboboxProps> = ({
```

Comment:
You should not use `React.FC`. Instead:
`export const VirtualizedCombobox: ({...}: VirtualizedComboboxProps) =>`